Makefile.in

```diff
@@ -757,7 +757,7 @@ stamps/build-gcc-newlib-stage2: $(GCC_SRCDIR) $(GCC_SRC_GIT) stamps/build-newlib
 		--prefix=$(INSTALL_DIR) \
 		--disable-shared \
 		--disable-threads \
-		--enable-languages=c,c++ \
+		--enable-languages=@WITH_LANGUAGES@ \
```

Comment:
As far as I can see, the changes here and in `configure.ac` are going to make the default set of languages `c,c++,fortran` for **all** toolchain variants whereas, right now, that's the case for Linux toolchains but bare-metal toolchains enable only `c,c++`.

Reply:
Do you want to keep this toolchain to `c,c++` instead of using macro?

Reply:
See my comment below:

- https://github.com/riscv-collab/riscv-gnu-toolchain/pull/1559#discussion_r1780062271